src/main/java/com/rtb/projectmanagementtool/user/UserData.java

```diff
@@ -1,7 +1,7 @@
 package com.rtb.projectmanagementtool.user;
 
 import com.google.appengine.api.datastore.Entity;
-import java.util.HashSet;
+import java.util.*;
```

Comment:
I would suggest listing out which specific utils you need from here.  I think java.util is huge and including all might not be efficient